### PR TITLE
feat: use readable resource names

### PR DESCRIPTION
**Pull request by OmarIthawi**
Automatically Transifex set resource names to a long name e.g. `translations..frontend-app-something..src-i18n-transifex-input--main`

This script sets it to `frontend-app-something` which makes it usable in the Transifex UI for translators.

Without this fix, it's almost not possible to know which resource belongs to which app.

Now the resource names look much better:

 - https://app.transifex.com/open-edx/openedx-translations/content/

<details><summary>Screenshot</summary>

image

</details>

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @OmarIthawi!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: .github/workflows/fix-transifex-resource-names.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This workflow runs fix-transifex-resource-names.py nightly
+
+name: Set the openedx-translations to readable resource names
+
+on:
+  workflow_dispatch:    # by request
+  schedule:
+    - cron: '0 0 * * *'  # every day at midnight
```
[the remark this answers is not in the document]
Comment (OmarIthawi) [reply .github/workflows/fix-transifex-resource-names.yml:8]: `weekly` or `monthly` are the alternatives that I have in mind? Taking a full week for new resources to be renamed might be too much.

Ideally this should run once for every new added resource in the `transifex.yml` file, but I'm not aware of a reliable way to do that.

That being said, changing it to `weekly` shouldn't be that bad.

[the remark this answers is not in the document]
Comment (brian-smith-tcril) [reply .github/workflows/fix-transifex-resource-names.yml]: > Ideally this should run once for every new added resource in the transifex.yml file, but I'm not aware of a reliable way to do that.

maybe there's a way to do this using
```yml
on:
    push:  
```
and [filters](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore)?

[the remark this answers is not in the document]
Comment (OmarIthawi) [reply .github/workflows/fix-transifex-resource-names.yml]: done.

**Review by brian-smith-tcril — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@OmarIthawi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-translations&entry.867055334=https://github.com/openedx/openedx-translations/pull/263&entry.1484655318=https://github.com/OmarIthawi&entry.752974735=2023-05-09+11:03&entry.1917517419=2023-05-24+14:33&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**